### DISCUSSION TITLE
scrollbar-props在data-table下不生效

### DISCUSSION THREAD

**Discussion by Zlinni** [tusen-ai/naive-ui#4890] [Q&A]
![image](https://github.com/tusen-ai/naive-ui/assets/78781776/d0a1af11-eff8-4313-97ff-91e4f0c9e2e1)
我这样注入scrollbarprops，但他在滚动的时候不生效

**Comment by jahnli** [comment]
@Zlinni 

https://github.com/tusen-ai/naive-ui/pull/4923

```
  <n-data-table
    :columns="columns"
    :data="data"
    :pagination="pagination"
    :max-height="300"
    @scroll="onScroll"
  />
```

**Comment by 07akioni** [comment]
使用 data-table 的 on-scroll 吧
